### PR TITLE
Support Outer Queries

### PR DESCRIPTION
**Pull request by 0aix**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/utils/query-converter.js [hunk 1]
```diff
@@ -77,9 +86,8 @@ const METRIC_TEST = new RegExp(`${METRIC}(?:,*${METRIC})*`, 'i');
 export default class QueryConverter {
   // BQL to Query methods
 
-  static classifyBQL(select, groupBy) {
-    // Group By
-    if (!isEmpty(groupBy)) {
+  static classifyBQL(select, groupBy, innerSelect, extraGroupBy) {
+    if (!isEmpty(extraGroupBy)) {
```
Comment (akshaisarma) [inline app/utils/query-converter.js:90]: extra is a good word for the things with a `2` 🤷

File: tests/unit/utils/query-converter-test.js [hunk 2]
```diff
@@ -656,4 +656,20 @@ module('Unit | Utility | query converter', function() {
       'SELECT * FROM STREAM(10000, TIME) WINDOWING EVERY(2000, TIME, ALL) LIMIT 10 '
     );
   });
+
+  test('it matches bql for a simple query correctly', function(assert) {
```
Comment (akshaisarma) [inline tests/unit/utils/query-converter-test.js:660]: Can we add more tests for complex nested queries with different query types?

**Review by akshaisarma — COMMENTED** [review]
Otherwise looks good

File: app/utils/query-converter.js (hunk 1 above)
Comment (0aix) [reply app/utils/query-converter.js:90]: going with "extra" in front of the regex group names. X instead of 2 for the regex const names.. I guess?

**Review by akshaisarma — APPROVED** [review]

**Pull request merged**